grafana/containers/pgbackrest.json

```diff
@@ -104,28 +188,41 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "ccp_backrest_last_incr_backup_time_since_completion_seconds{pg_cluster=\"[[cluster]]\", role=\"master\"}",
+          "expr": "min(ccp_backrest_last_incr_backup_time_since_completion_seconds{pg_cluster=\"[[cluster]]\", role=\"master\"}) without(deployment,instance,ip,pod)",
```

Comment:
Curious why you are doing a min() here? I don't believe you should be getting back multiple results from this for a single cluster. Same for other places where you do a min().

Reply:
In the K8s env, as pods restart, they change their name. This approach provides continuity to the graphs without duplicates

Reply:
Ok, that makes sense.